charts/datahub/subcharts/datahub-mae-consumer/templates/deployment.yaml

```diff
@@ -61,7 +61,8 @@ spec:
         - name: {{ .Chart.Name }}
           securityContext:
             {{- toYaml .Values.securityContext | nindent 12 }}
-          image: "{{ .Values.image.repository }}:{{ required "Global or specific tag is required" (.Values.image.tag | default .Values.global.datahub.version) }}"
+          {{- $registry := .Values.global.imageRegistry | default .Values.image.registry -}}
```

Comment:
Can we add this as a helper in `charts/datahub/templates/_helpers.tpl` instead of duplicating the logic across many templates?

Reply:
Thanks for the feedback. I added the usage of templates. Unfortunately especially the datahub-upgrade jobs don't use the same mechanisms and structuring like subcharts so I simply duplicated the templates.
One idea would be to re-map the variables like a context but I am not familiar with it yet and do not want to blow up this PR more.
But would be a refactoring to do after that?

Reply:
Looking better. I provided an example that removes the need to duplicate templates.